lib/collection.js

```diff
@@ -746,7 +748,7 @@ Collection.prototype.updateOne = function(filter, update, options, callback) {
  * @param {boolean} [options.bypassDocumentValidation=false] Allow driver to bypass schema validation in MongoDB 3.2 or higher.
  * @param {ClientSession} [options.session] optional session to use for this operation
  * @param {Collection~updateWriteOpCallback} [callback] The command result callback
- * @return {Promise} returns Promise if no callback passed
+ * @return {Promise} returns Promise resolving to {Collection~updateWriteOpResultObject} if no callback passed
```

Comment:
@dandv Could we instead describe the type as `Promise<Collection~updatewriteOpResultObject>`?